docs/getting-started/configuration.md

```diff
@@ -66,7 +66,7 @@ Stop Backstage, and go to the root directory of your freshly installed Backstage
 App. Use the following commands to start the PostgreSQL client installation:
 
 ```bash
-# From your Backstage root directory
+# From your Backstage app root directory
```

Comment:
```suggestion
# From your freshly created Backstage root directory
```

Maybe? Since "app" often is used to refer to packages/app

Reply:
Could refer to the "workspace root" as well since that's the Yarn terminology